source/monitoring/cluster-monitoring.txt

```diff
@@ -0,0 +1,249 @@
+
+.. literalinclude:: /includes/monitoring/monitoring.rb
+   :start-after: start-sdam
+   :end-before: end-sdam
+   :language: ruby
+   :copyable:
+   :dedent:
+
+To subscribe to events, you need to create the appropriate subscriber and 
+subscribe to the correct monitoring ``topic``. The following code shows how 
+to subscribe to SDAM events globally:
+
+.. literalinclude:: /includes/monitoring/monitoring.rb
+   :start-after: start-sdam-subscriber-global
+   :end-before: end-sdam-subscriber-global
+   :language: ruby
+   :copyable:
+   :dedent:
+
+The following code shows how to subscribe to SDAM events for a single client 
+by using the ``sdam-proc`` client option:
+
+.. literalinclude:: /includes/monitoring/monitoring.rb
+   :start-after: start-sdam-subscriber-client
+   :end-before: end-sdam-subscriber-client
+   :language: ruby
+   :copyable:
+   :dedent:
+
+When you run the application, your subscriber records the SDAM event and 
+outputs messages such as the following:
+
+.. code-block:: none
+
+  D, [2018-10-09T13:58:03.489461 #22079] DEBUG -- : SDAM | Topology type 'Unknown' initializing.
+  D, [2018-10-09T13:58:03.489699 #22079] DEBUG -- : SDAM | Server 127.0.0.1:27100 initializing.
+  D, [2018-10-09T13:58:03.491384 #22079] DEBUG -- : SDAM | Server description for 127.0.0.1:27100 changed from 'unknown' to 'unknown'.
+  D, [2018-10-09T13:58:03.491642 #22079] DEBUG -- : SDAM | Server localhost:27100 initializing.
+  D, [2018-10-09T13:58:03.493199 #22079] DEBUG -- : SDAM | Server description for localhost:27100 changed from 'unknown' to 'primary'.
+  D, [2018-10-09T13:58:03.493473 #22079] DEBUG -- : SDAM | Server localhost:27101 initializing.
+  D, [2018-10-09T13:58:03.494874 #22079] DEBUG -- : SDAM | Server description for localhost:27101 changed from 'unknown' to 'secondary'.
+  D, [2018-10-09T13:58:03.495139 #22079] DEBUG -- : SDAM | Server localhost:27102 initializing.
+  D, [2018-10-09T13:58:03.496504 #22079] DEBUG -- : SDAM | Server description for localhost:27102 changed from 'unknown' to 'secondary'.
+  D, [2018-10-09T13:58:03.496777 #22079] DEBUG -- : SDAM | Topology type 'Unknown' changed to type 'ReplicaSetNoPrimary'.
+  D, [2018-10-09T13:58:03.497306 #22079] DEBUG -- : SDAM | Server 127.0.0.1:27100 connection closed.
+  D, [2018-10-09T13:58:03.497606 #22079] DEBUG -- : SDAM | Topology type 'ReplicaSetNoPrimary' changed to type 'ReplicaSetWithPrimary'.
+
+  # client.close
+
+  D, [2018-10-09T13:58:05.342057 #22079] DEBUG -- : SDAM | Server localhost:27100 connection closed.
+  D, [2018-10-09T13:58:05.342299 #22079] DEBUG -- : SDAM | Server localhost:27101 connection closed.
+  D, [2018-10-09T13:58:05.342565 #22079] DEBUG -- : SDAM | Server localhost:27102 connection closed.
+  D, [2018-10-09T13:58:05.342693 #22079] DEBUG -- : SDAM | Topology type 'ReplicaSetWithPrimary' closed.
+
+.. note::
+
+  The ``:sdam_proc`` client option applies only to the given client. 
+  When certain client options are changed via the
```

Comment:
S: to avoid [via](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#v):
```suggestion
  When certain client options are changed by using the
```